src/olympia/discovery/admin.py

```diff
@@ -9,14 +9,18 @@
 from django.utils.safestring import mark_safe
 from django.utils.translation import gettext
 
-from olympia import promoted
+from olympia import hero, promoted
 from olympia.addons.models import Addon
 from olympia.amo.admin import AMOModelAdmin
 from olympia.amo.templatetags.jinja_helpers import vite_asset
 from olympia.discovery.models import DiscoveryItem
```

Comment:
Is all of this really necessary for this PR? From what I can see this is a convenience? Or am I missing something? If, so I'd cut the 200 lines from the diff.

I'm really not trying to be a kill joy, just we have had several bugs during the roll out of this feature and the smaller the PR the less likely it is something goes wrong.